templates/cloud/index.html

```diff
@@ -92,56 +120,122 @@
-        <li class="p-list__item is-ticked">Includes management tooling and commercial support from Canonical</li>
+        <li class="p-list__item is-ticked">Jointly shape the architecture to meet your requirements</li>
+        <li class="p-list__item is-ticked">We build OpenStack in your data center of choice, and operate it to an SLA</li>
+        <li class="p-list__item is-ticked">When your team is ready to take on OpenStack, we’ll hand over the keys</li>
+      </ul>
+      <p><a href="/cloud/managed-cloud">Let Canonical build, operate and transfer your OpenStack&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col-6">
+      <ul class="p-inline-images">
+        <li class="p-inline-images__item">
+          <img src="{{ ASSET_SERVER_URL }}01ed776c-veritas.svg" alt="Veritas" />
+        </li>
+        <li class="p-inline-images__item">
+          <img src="{{ ASSET_SERVER_URL }}66bc9135-logo-lexisnexis.png" alt="Lexis Nexis" />
+        </li>
+        <li class="p-inline-images__item">
+          <img src="{{ ASSET_SERVER_URL }}25de1877-Tele2.svg" alt="Tele2" />
+        </li>
+        <li class="p-inline-images__item">
+          <img src="{{ ASSET_SERVER_URL }}d0b7aa28-logo-deutsche-telekom.png" alt="Deutsche Telekom" />
+        </li>
+        <li class="p-inline-images__item">
+          <img src="{{ ASSET_SERVER_URL }}481063cd-pccw.svg" alt="PCCW logo" />
+        </li>
+        <li class="p-inline-images__item">
+          <img src="{{ ASSET_SERVER_URL }}d56d0b96-Biznet-Gio.png" alt="Biznet Gio logo" />
+        </li>
+        <li class="p-inline-images__item">
+          <img src="{{ ASSET_SERVER_URL }}ff4e18ac-mimedia_logo_lrg.png" alt="Mimedia logo" />
+        </li>
+        <li class="p-inline-images__item">
+          <img src="{{ ASSET_SERVER_URL }}507a0979-Voidbridge.jpg" alt="Voidbridge" />
+        </li>
+        <li class="p-inline-images__item">
+          <img src="{{ ASSET_SERVER_URL }}2e38f398-uct.svg" alt="UCT" />
+        </li>
       </ul>
-      <p><a href="/download/cloud/autopilot">Get OpenStack Autopilot&nbsp;&rsaquo;</a></p>
     </div>
   </div>
 </section>
 
-<section class="p-strip--dark is-deep">
+<section class="p-strip--image u-no-background--small is-deep is-bordered" style="background-image: url('https://assets.ubuntu.com/v1/1d2ab6ba-suru-background.png'); background-size: 100% 100%;">
+  <div class="row">
+    <div class="col-8">
+      <h2>Foundation Cloud Build for Ubuntu OpenStack</h2>
+      <p>We will build you a highly available production cloud, implemented on-site with Canonical&rsquo;s expert delivery team in the shortest possible time.</p>
+      <p>Foundation Cloud Build makes redeploying or cloning your cloud architecture easy and affordable. We will also handle any important changes to your infrastructure including config modifications, OS and platform version upgrades, and deeper design revisions.</p>
```

Comment:
Content here doesn't match the copydoc

Reply:
Done, thanks.

Reply:
this is still missing the CTA [Find out more about Foundation Cloud Build >](https://www.ubuntu.com/cloud/foundation-cloud)

Reply:
Good spot, thanks. Fixed